doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc

```diff
@@ -145,6 +145,133 @@ $ oc create -f cfme-restore-job.yaml
+
+[IMPORTANT]
+====
+If you are updating a {product-title} 4.6.0 environment to version 4.6.1, follow the steps in xref:modified-template-update[]. When updating between these versions, the template must also be updated to reflect the updated ConfigMap mount path for `postgresql`. 
+====
+endif::cfme[]
+
+ifdef::miq[]
+[IMPORTANT]
+====
+If you are updating a {product-title} 5.9.0 environment to version 5.9.1, follow the steps in xref:modified-template-update[]. When updating between these versions, the template must also be updated to reflect the updated ConfigMap mount path for `postgresql`.
+====
+endif::miq[]
+
+
+[[image-update]]
+==== Updating the {product-title_short} Container Images
+
+To deploy new container images for {product-title_short}, delete the pods, update the images, then rebuild the updated pods:
+
+. Scale the application StatefulSets to 0 replicas:
++
+----
+$ oc scale statefulset cloudforms --replicas=0
+$ oc scale statefulset cloudforms-backend --replicas=0
+----
++
+. Patch in the new version of the images:
++
+----
+$ oc patch statefulset cloudforms -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app-ui:<new-version-tag>"}]}}}}'
+$ oc patch statefulset cloudforms-backend -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app:<new-version-tag>"}]}}}}'
+----
++
+. Scale the StatefulSets back to your previous values to redeploy the pods:
++
+----
+$ oc scale statefulset cloudforms --replicas=<old-value>
+$ oc scale statefulset cloudforms-backend --replicas=<old-value>
+----
+
+Your {product-title_short} environment now updated.
+
+
+[[modified-template-update]]
+==== Updating the {product-title_short} Container Images and Template
+
+To update a {product-title_short} deployment with a modified template, update the container images and the template using the following steps:
+
+. Back up secrets, such as database encryption keys and credentials:
++
+[IMPORTANT]
+====
+Be careful to back up secrets in a secure location.
+====
++
+----
+$ oc export secret cloudforms-secrets > my_secrets.yml
+----
++
+. Scale the application StatefulSets to 0 replicas:
++
+----
+$ oc scale statefulset cloudforms --replicas=0
+$ oc scale statefulset cloudforms-backend --replicas=0
+----
++
+. Patch in the new version of the images:
++
+----
+$ oc patch statefulset cloudforms -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app-ui:<new-version-tag>"}]}}}}'
+$ oc patch statefulset cloudforms-backend -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app:<new-version-tag>"}]}}}}'
```

Comment:
This step is not necessary here because the new image version will be built into the new template.

Reply:
Also if the user wants to use a version different from the default in the new template, they can set that using the parameters in the `oc process` command.

Reply:
Ah cool - I've deleted this step now.